### PR TITLE
Feature/61 consitent button styling

### PR DESCRIPTION
**Pull request by gonzalezpear8**
## Description

Changed the CSS files and formatting of the links/buttons/dropdown boxes/text fields
## How has this been tested?
This was tested by running the server locally and seeing the changes. 

## Screenshots (if appropriate):
image
image
image
image
image
image
image

## Checklist:

- [x] Zero `rubocop` violations
- [x] My code receives an "A" when `rubycritic` is run
- [x] >90% test coverage of my changes
- [x] All new and existing tests passed

### CODE REVIEW

File: app/views/assignments/_assignment.html.erb [hunk 1]
```diff
@@ -1,12 +1,11 @@
-<div id="<%= dom_id assignment %>">
+<div id="<%= dom_id assignment %>" class="assignment-item">
   <p>
-    <strong>Assignment name:</strong>
-    <%= assignment.assignment_name %>
+    <strong>Assignment dsafdafdasfd:</strong>
```
Comment (maxsmith271346) [inline app/views/assignments/_assignment.html.erb:3]: Please change this to `Assignment Name`

**Review by maxsmith271346 — CHANGES_REQUESTED** [review]

**Review by maxsmith271346 — APPROVED** [review]

**Review by maxsmith271346 — CHANGES_REQUESTED** [review]
There is a `.feature` file associated with this feature.  Please use `git checkout <branch> <filename>` to get this into your branch, then write the necessary step definitions.

**Comment by riddhighate-tamu** [comment]
For the consistent styling feature, go to `sprint2_features` branch. There is a file called `features/consistent_styling.feature`, only pull this file in your branch and complete the scenarios for it.

**Comment by maxsmith271346** [comment]
@gonzalezpear8 Is there any update on this?

**Comment by riddhighate-tamu** [comment]
Can you push the feature file and the related cucumber steps?

**Comment by riddhighate-tamu** [comment]
Please take pull from the `main` branch and resolve the conflicts.

File: app/views/assignments/show.html.erb [hunk 2]
```diff
@@ -74,6 +74,13 @@
       <div id="test-form" class="test-form">
         <h2><%= @test.persisted? ? "Edit Test for Assignment #{@assignment.assignment_name}" : "New Test for Assignment #{@assignment.assignment_name}" %></h2>
 
+        <% if @test.persisted? %>
+          <div class="delete-actions">
+            <%= button_to "Destroy this test", assignment_test_path(@assignment, @test), method: :delete, data: { confirm: "Are you sure?", turbo: false }, class: "btn btn-danger" %>
```
Comment (maxsmith271346) [inline app/views/assignments/show.html.erb:79]: Please change this to say "Delete Test"

**Review by maxsmith271346 — CHANGES_REQUESTED** [review]

**Review by riddhighate-tamu — APPROVED** [review]

**Comment by gonzalezpear8** [comment]
image

**Review by masarkar-tamu — APPROVED** [review]

**Pull request merged**